cico_build_deploy.sh

```diff
@@ -39,16 +39,16 @@ echo 'CICO: Deploying to github pages'
 git fetch origin gh-pages
 git checkout -b gh-pages origin/gh-pages
 
-# Move over generated content to new repo 
+# Move over generated content to new repo
 cp -rfv _site/* .
 
 # Update new repo with generated code
 git config user.name "ci.centos.org"
-git config user.email "almighty-public@redhat.com"
+git config user.email "fabric8-services-public@redhat.com"
```

Comment:
where did "fabric8-services-public@redhat.com" come from ?

there is fabric8@googlegroups.com and fabric8-planner@googlegroups.com...why now a fabric8 redhat.com one ?